extensionsGallery-insider.json

```diff
@@ -3196,14 +3196,14 @@
 					},
 					"versions": [
 						{
-							"version": "0.2.0",
-							"lastUpdated": "06/01/2020",
+							"version": "0.2.1",
```

Comment:
I guess it doesn't really matter too much - but why 0.2.1 and not just 0.3.0? Typically our extensions have always only updated the Major/Minor version so it'd be nice to keep things consistent with the others IMO

Reply:
https://semver.org/
MAJOR version when you make incompatible API changes,
MINOR version when you add functionality in a backwards compatible manner, and
PATCH version when you make backwards compatible bug fixes.

i guess 0.2.1 seems to be fair here

Reply:
0.2.0 was for functionality change (added support for MI) and since that was already shiped in insider, I updated to 0.2.1 for one small bug I fixed